buildMirge.sh

```diff
@@ -53,10 +53,19 @@ fi
 
 if [ -z ${CONDA_PATH+x} ]; then
   echo "CONDA_PATH unset, installing new conda with emirge"
-  echo "./install.sh --env-name=mirgeDriver.Y1nozzle $git_method --branch=y1-production"
-  ./install.sh --env-name=mirgeDriver.Y1nozzle $git_method --branch=y1-production
+  echo "./install.sh --env-name=mirgeDriver.Y1nozzle $git_method --branch=parallel-lazy"
+  ./install.sh --env-name=mirgeDriver.Y1nozzle $git_method --branch=parallel-lazy
 else
   echo "Using existing Conda installation, ${CONDA_PATH}"
-  echo "./install.sh --conda-prefix=$CONDA_PATH --env-name=mirgeDriver.Y1nozzle $git_method --branch=y1-production"
-  ./install.sh --conda-prefix=$CONDA_PATH --env-name=mirgeDriver.Y1nozzle $git_method --branch=y1-production
+  echo "./install.sh --conda-prefix=$CONDA_PATH --env-name=mirgeDriver.Y1nozzle $git_method --branch=parallel-lazy"
+  ./install.sh --conda-prefix=$CONDA_PATH --env-name=mirgeDriver.Y1nozzle $git_method --branch=parallel-lazy
```

Comment:
This should stay y1-production?

Reply:
Not sure how to make that work.  Any ideas?

Reply:
OIC. we need the parallel lazy branch to get the correct versions of the sub-packages from the emirge install.

Reply:
Yeah, one idea is to just merge this into y1-production just like we do for NS and AV.  Any objections @matthiasdiener?